locale/en/blog/weekly-updates/weekly-update.2016-01-22.md

```diff
@@ -0,0 +1,55 @@
+  - **[@jaredhanson](https://github.com/jaredhanson)**: [nomination post](http://jaredhanson.net/blog/2016/01/13/im-running-for-the-node-js-foundation-bod/), [Q/A thread](https://github.com/nodejs/membership/issues/24)
+  - **[@tejasmanohar](https://github.com/tejasmanohar)**: [nomination post](https://medium.com/@tejasmanohar/node-js-foundation-board-of-directors-5514e8faa660), [Q/A thread](https://github.com/nodejs/membership/issues/25)
+  - **[@bnb](https://github.com/bnb)**: [nomination post](http://bnb.im/blog/Individual-Membership-on-the-Board-of-Directors-for-Node-js.html), [Q/A thread](https://github.com/nodejs/membership/issues/26)
+  - **[@watson](https://github.com/watson)**: [nomination post](https://medium.com/@wa7son/why-i-m-running-for-the-node-js-foundation-board-of-directors-253bc2e3a834), [Q/A thread](https://github.com/nodejs/membership/issues/27)
+  - **[@feross](https://github.com/feross)**: [nomination post](http://feross.org/node-board/), [Q/A thread](https://github.com/nodejs/membership/issues/28)
+  - **[@pgte](https://github.com/pgte)**: [nomination post](https://gist.github.com/pgte/cfbf468202b35be78c66), [Q/A thread](https://github.com/nodejs/membership/issues/29)
+  - **[@ecowden](https://github.com/ecowden)**: [nomination post](https://medium.com/@evan.cowden/the-world-s-worst-resume-e0adf234baa0), [Q/A thread](https://github.com/nodejs/membership/issues/30)
+
+A ballot will be distributed to individual members on January 20th, with the election completed by January 30th.
+
+To be eligible to vote, you must be signed up as an individual member of the Node.js Foundation, more information can be found here: https://nodejs.org/en/blog/community/individual-membership/
+
+### Nodesource Updates
+
+* [Node by Numbers 2015](https://nodesource.com/blog/node-by-numbers-2015/), "By analyzing the logs for binary and source downloads, we get to discover some interesting patterns."
+* [The (Enterprise) Node Community](https://nodesource.com/blog/the-enterprise-node-community/), "At NodeSource, we believe that by embracing the greater Node.js community of developers, operations folks and even executives in the enterprise can radically expand our community."
+* [N|Solid v1.2 - monitoring Node.js at scale](https://nodesource.com/blog/n-solid-v1-2-monitoring-node-js-at-scale/), "N|Solid v1.2 adds new functionality that greatly simplifies monitoring Node at scale across your environment."
+
+### Nodeup Updates
+
+* [ninetyseven](http://nodeup.com/ninetyseven), "A Front-end npm Show"
+
+### Community Updates
+
+* [How to track down CPU issues in Node.js](http://apmblog.dynatrace.com/2016/01/14/how-to-track-down-cpu-issues-in-node-js/), "CPU aspects of Node.js including the event loop and how to track down tasks that consume too much CPU time and slow down the whole process."
```

Comment:
I think the advertising is at a benign capacity, it's not to much, you know. 
> Where do we draw the line

If I'm honest, we should open an issue & reference this. And let it age a bit, because people have
opinions for days, but i do think your right, in bringing some awareness to it. :smiley: